bitnami/fluentd/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and .Values.serviceAccount.create .Values.rbac.create }}
```

Comment:
Example of incomplete values for the validation. You cannot have rbac create without serviceAccount create